js/everything.js

```diff
@@ -378,20 +378,21 @@ function toggleElement(event, toggleID, buttonText) {
  */
 function getCurrentVideoID() {
     var v = URI(window.location).search(true).v;
-    if (!v) {
-        return null;
-    }
-
-    if (Array.isArray(v)) {
-        v = v[0];
+    var r;
+    if (v) {
```

Comment:
This appears to be an issue with Github, the code in file is;
`function getCurrentVideoID() {
    var v = URI(window.location).search(true).v;
    var r;
    if (v) {
        if (Array.isArray(v)) {
            r = wrapParseYouTubeVideoID(v.pop());
        } else if (v) {
            r = wrapParseYouTubeVideoID(v);
        }
    }

    return r;
}`